desktop/hardened-desktop/enhanced-container-isolation/index.md

```diff
@@ -0,0 +1,98 @@
+- For organizations that want to ensure stronger container isolation that is easy and intuitive to implement on developers' machines.
+
+### What happens when Enhanced Container Isolation is enabled?
+
+When Enhanced Container Isolation is enabled using [Settings Management](../settings-management/index.md), the following features are enabled: 
+
+- All user containers are automatically run in Linux User Namespaces which ensures stronger isolation.
+- The root user in the container maps to an unprivileged user at VM level.
+- Users can continue using containers as usual, including bind-mounting host directories, volumes, networking configurations, etc.
+- Privileged containers work, but they are only privileged within the container's Linux User Namespace, not in the Docker Desktop VM.
+- Containers can no longer share namespaces with the Docker Desktop VM. For example, `--network=host`, `--pid=host`.
+- Containers can no longer modify configuration files in the Docker Desktop VM.
+- Console access to the Desktop VM is forbidden for all users.
+- Containers become harder to breach. For example, sensitive system calls are vetted and portions of `/proc` and `/sys` are emulated.
+
+For more information on how Enhanced Container Isolation work, see [How does it work](how-eci-works.md).
+
+>**Important**
+>
+>Enhanced Container Isolation is currently incompatible with WSL and does not protect Kubernetes pods. For more information on known limitations and workarounds, see [FAQs and known issues](faq.md).
+{: .important}
+
+### How do I enable Enhanced Container Isolation?
+
+As an admin, you first need to [configure a `registry.json` file to enforce sign-in](../../../docker-hub/configure-sign-in.md). This is because the Enhanced Container Isolation feature requires a Docker Business subscription and therefore your Docker Desktop users must authenticate to your organization for this configuration to take effect.
+
+Next, you must [create and configure the `admin-settings.json` file](../settings-management/configure.md) and specify:
+
+```JSON
+{
+ "enhancedContainerIsolation": {
+    "value": true,
+    "locked": true
+    }
+}
+```
+
+Once this is done, developers need to either quit, re-launch, and sign in to Docker Desktop, or launch and sign in to Docker Desktop for the first time.
+
+>Important
+  >
+  >Selecting **Restart** from the Docker menu isn't enough as it only restarts some components of Docker Desktop.
+  {: .important}
+
+### What do users see when this setting is enforced?
+
+When Enhanced Container Isolation is enabled, users see that containers run within a Linux user-namespace. 
+
+To check, run:
+
+```
+$ docker run -it --rm alpine / # cat /proc/self/uid_map 
+```
+
+The following output displays:
+
+```
+         0     100000      65536
+```
+
+This indicates that the container's root user (0) maps to unprivileged user (100000) in the Docker Desktop VM, and that the mapping extends for a range of 64K user-IDs.
```

Comment:
After this sentence please add:

"If you create more containers, the user-ID mapping will vary as each container gets an exclusive range of host User-IDs for isolation. This mapping automatically managed by Docker Desktop."